site/resources.md

```diff
@@ -6,33 +6,63 @@ id: resources
 ---
 Here you will find external resources about Velero, such as videos, podcasts, and community articles.
 
-## Previous community meetings
+## All community meetings
 
 <iframe title="Velero Community Meetings/Open Discussions" width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Podcast shows
 
+* vSpeaking Podcast Ep 124, with guests Carlisia Campos, Myles Gray and Cormac Hogan: https://blogs.vmware.com/virtualblocks/2019/08/02/vspeaking-podcast-velero/
+
 * Listen to the PodCTL podcast with Carlisia Campos talking about Velero: <http://podcast.podctl.com/110399/986641-understanding-project-velero-formerly-ark>
 
 ## Videos
 
+* Kubecon NA 2019 - How to Backup and Restore Your Kubernetes Cluster - Annette Clewett & Dylan Murray, Red Hat:
+
+<iframe title="How to Backup and Restore Your Kubernetes Cluster - Annette Clewett & Dylan Murray, Red Hat" width="560" height="315" src="https://www.youtube.com/embed/JyzgS-KKuoo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
+* Kubernetes Back Up, Restore and Migration with Velero - A NewStack interview, with guests: Carlisia Campos, Tom Spoonemore, and Efri Nattel-Shayo:
```

Comment:
If you want to keep the `</br>`, make sure there's a line between it and the next bullet, otherwise it will just render as an asterisk.